arm-wt-22k/lib_src/eas_voicemgt.c

```diff
@@ -307,8 +325,36 @@ EAS_RESULT VMInitialize (S_EAS_DATA *pEASData)
     pSecondarySynth->pfInitialize(pVoiceMgr);
 #endif
 
+#ifdef _CC_CHORUS
+    result = EAS_Chorus.pfInit(pEASData, &pVoiceMgr->chorusData);
+    if (result != EAS_SUCCESS) {
+        EAS_Report(_EAS_SEVERITY_FATAL, "VMInitialize: EAS_Chorus.pfInit returned %ld\n", result);
+        goto error_cleanup;
+    }
+    EAS_Chorus.pFSetParam(pVoiceMgr->chorusData, EAS_PARAM_CHORUS_BYPASS, EAS_FALSE);
+    EAS_Chorus.pFSetParam(pVoiceMgr->chorusData, EAS_PARAM_CHORUS_DRY, 0);
+    EAS_Chorus.pFSetParam(pVoiceMgr->chorusData, EAS_PARAM_CHORUS_LEVEL, EAS_CHORUS_LEVEL_MAX);
+#endif
+
+#ifdef _CC_REVERB
+    result = EAS_Reverb.pfInit(pEASData, &pVoiceMgr->reverbData);
```

Comment:
```suggestion
    pEAS_Reverb = pEASData->effectsModules[EAS_MODULE_REVERB].effect;
    if (pEAS_Reverb) {
        result = pEAS_Reverb->pfInit(pEASData, &pVoiceMgr->reverbData);
```
And replace `EAS_Reverb.` by `pEAS_Reverb->` in the following lines.